Libraries/LibWeb/HTML/CanvasRenderingContext2D.h

```diff
@@ -78,7 +80,7 @@ class CanvasRenderingContext2D
 
     virtual WebIDL::ExceptionOr<GC::Ref<ImageData>> create_image_data(int width, int height, Optional<ImageDataSettings> const& settings = {}) const override;
     virtual WebIDL::ExceptionOr<GC::Ref<ImageData>> create_image_data(ImageData const& image_data) const override;
-    virtual WebIDL::ExceptionOr<GC::Ptr<ImageData>> get_image_data(int x, int y, int width, int height, Optional<ImageDataSettings> const& settings = {}) const override;
+    virtual WebIDL::ExceptionOr<GC::Ptr<ImageData>> get_image_data(int x, int y, int width, int height, Optional<ImageDataSettings> const& settings = {}) override;
```

Comment:
Is it really necessary not to keep it const?

Reply:
It is because draw_bitmap wasn't const anymore because of the list of filters but I can try to pass them as a const Span

Reply:
It would be nice to try that. Keeping it const might prevent a getter from accidentally modifying something in the future, reducing the risk of making subsequent changes more error-prone